rmw_connext_shared_cpp/include/rmw_connext_shared_cpp/wait.hpp

```diff
@@ -79,10 +168,7 @@ wait(
       }
 
       for (DDS::Long i = 0; i < attached_conditions->length(); ++i) {
-        retcode = dds_wait_set->detach_condition((*attached_conditions)[i]);
-        if (retcode != DDS::RETCODE_OK) {
-          RMW_SET_ERROR_MSG("Failed to get detach condition from wait set");
-        }
+        __detach_condition(dds_wait_set, (*attached_conditions)[i]);
```

Comment:
You're ignoring the return code of this implicitly, is that intentional? If so, explicitly ignoring it is better (especially with a comment as to why).

Reply:
Fixed, added the error back.